requirements.txt

```diff
@@ -1,3 +1,4 @@
 boto3~=1.9.246
 kubernetes~=10.0.1
-python-dotenv~=0.10.2
\ No newline at end of file
+python-dotenv~=0.10.2
+threaded~=4.1.0
```

Comment:
Are you actually using [threaded](https://pypi.org/project/threaded/)? Not seeing any decorators used in your code, unless it's a transitive dependency?

Reply:
Nah, this was added by my miss understanding. I was under the impression that threaded was required for threading. But just found out threading is part of the standard library. Thanks!